src/compositemark/errorbar.ts

```diff
@@ -512,26 +512,26 @@ function errorBarAggregationAndCalculation<
     if (inputType === 'aggregated-upper-lower') {
       tooltipSummary = [];
       postAggregateCalculates = [
-        {calculate: `datum.${continuousAxisChannelDef2.field}`, as: `upper_` + continuousFieldName},
-        {calculate: `datum.${continuousFieldName}`, as: `lower_` + continuousFieldName}
+        {calculate: `datum["${continuousAxisChannelDef2.field}"]`, as: `upper_` + continuousFieldName},
```

Comment:
Please combine the strings into one template string.

Reply:
For consistency with `as` clauses in other transforms in composite mark, should we use `'upper_' + continuousFieldName`?

Reply:
I don't understand the question.